shared/styles/index.native.js

```diff
@@ -40,6 +40,7 @@ const fontIOS = {
 const fontAndroid = {
   fontRegular: {
     fontFamily: 'OpenSans',
+    fontWeight: 'normal',
```

Comment:
annoying it doesn't default to this. Do we have to do this for the other fonts that don't have an explicit weight?